sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -28,12 +31,13 @@
 import tempfile
 import time
 from datetime import datetime
-from StringIO import StringIO
+from io import BytesIO
```

Comment:
In some of your other pending changes you use `io.BytesIO`.  Can we choose one style and be consistent?

Reply:
Are there any guidelines on the style? If not, I'll make it consistent with the `io.BytesIO`

Reply:
I think people usually use it in a qualified way.

Reply:
Check!